.github/workflows/main.yml

```diff
@@ -10,12 +10,11 @@ jobs:
       fail-fast: false
       matrix:
         node-version:
-          - 16
-          - 14
-          - 12
+          - 20
+          - 18
```

Comment:
Node 18 is already LTS: https://nodejs.org/en

Reply:
But 16 is also still LTS, and I generally prefer to support all active LTS versions.